.github/workflows/Arduino-CI.yaml

```diff
@@ -20,6 +20,12 @@ jobs:
       - run: |
           bundle install
 
+      # Creates the Library directory and install our libraries
+      - run: |
+          bundle exec ensure_arduino_installation.rb
```

Comment:
Is this needed for running tests locally or only for the CI?

Reply:
Yes, we need to install custom libraries so it does make sense to have an install script that is called by the testing framework and is the one we can use as well.